source/reference/permissions.txt

```diff
@@ -0,0 +1,27 @@
+.. _c2c-permissions-and-roles:
+
+================
+User Permissions
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/fact-permissions-body.rst
+
+
+Self-Managed Clusters
+---------------------
+
+.. include:: /includes/table-permissions-self-hosted.rst
+
+Atlas Clusters
+--------------
+
```

Comment:
**Suggestion**

Add a small blurb, 

Ex: Atlas Clusters 

Atlas Clusters require an additional bypassWriteBlockMode privilege for all write-blocking or reversing sync types.